### PR TITLE
Adjust name and volume in templates

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

### CODE REVIEW

File: openshift/job-template.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ metadata:
     template.openshift.io/provider-display-name: "Red Hat, Inc."
     thoth-station.ninja/template-version: 0.10.0
   labels:
-    template: solver
+    template: solver-workload-operator
```
Comment (goern) [inline openshift/job-template.yaml]: does this renaming need to be reflected somewhere in thoth.common?

Comment (pacospace) [reply openshift/job-template.yaml]: I already addressed it for all component in thoth-common :) for solver for example https://github.com/thoth-station/common/blob/0635aac04e7331defcb13d0d3223570290370223/thoth/common/openshift.py#L940

**Comment by fridex** [comment]
recheck

**Comment by fridex** [comment]
regate

**Pull request merged**